packages/fast-components-react-base/src/breadcrumb/README.md

```diff
@@ -0,0 +1,8 @@
+## Breadcrumb
+A generic breadcrumb component following the w3.org [breadcrumb](https://www.w3.org/TR/2017/NOTE-wai-aria-practices-1.1-20171214/examples/breadcrumb/index.html) pattern. The *breadcrumb* will accept any child elements but is built to work with *breadcrumb-item* component.
+
+### Usage
+The *breadcrumb* automatically sets the last *breadcrumb-item* `href` to "undefined" and `current` prop to "true". Addtionally *breadcrumb* accepts an optional `seperator` prop as a JSX.element that will render a JSX element between navigation elements. To support `rtl` in jss you need to target the `ol` in *breadcrumb* at set it to `display: "flex", flexWrap: "wrap"`

```

Comment:
Can you provide a why for "...automatically sets the last *breadcrumb-item* `href` to "undefined" and `current` prop to "true"."

Reply:
And, do we not support RTL out of the box here because it is base?